pkg/services/cosmosdb/catalog.go

```diff
@@ -9,13 +9,13 @@ func (m *module) GetCatalog() (service.Catalog, error) {
 			service.NewService(
 				&service.ServiceProperties{
 					ID:   "6330de6f-a561-43ea-a15e-b99f44d183e6",
-					Name: "azure-cosmos-document-db",
-					Description: "Azure DocumentDB (Experimental) provided by CosmosDB " +
-						"and accessible via SQL (DocumentDB), Gremlin (Graph), and Table " +
-						"(Key-Value) APIs",
+					Name: "azure-cosmos-db",
+					Description: "Azure Cosmos DB is a globally distributed, " +
+						"multi-model database service accessible via SQL, " +
+						"Gremlin (Graph), and Table (Key-Value) APIs",
```

Comment:
Should we change the name and description to be clear that what this provisions is an _account_?

Reply:
Yeah, good point. Will update that.

Reply:
@jeremyrickard should we consider changing the name of the manager to `cosmosAccountManager`? Presumably, there are database and/or collection managers (not totally clear yet) that will be added in a near-future PR.

Reply:
Along those lines, do you think we'll add multiple account managers?

Reply:
Probably so.

Reply:
I left the _names_ the same for now, but cleaned up the the description to indicate A.) It's an account only and B, it's the specific API (either SQL or MongoDB). Will add the other APIs in another PR and if we do one to provision a collection on the account, will rename